docs/it/stime.md

```diff
@@ -0,0 +1,38 @@
+#### Comprendere il processo
+
+Quando si parla di stima si intende riuscire a delineare un confine temporale per la realizzazione di un'opera di qualche genere. Ci troviamo di fronte ad un tema controverso, chiedendo a dieci persone probabile che otterremo dieci risposte diverse con metodi o approcci anche molto distanti. Qui non vogliamo andare nel profondo di uno o più metodi ma chiarire il tema per comprenderlo e dare degli strumenti e spunti di approfondimento.
+
+Nell'ambito informatico, la stima, può essere usata per dare una dimensione ad un task o ad un intero progetto, aiuta il business a delineare un'idea o a pianificare le attività ed è quindi piuttosto importante perché consente di prendere decisioni su come spendere le risorse; dobbiamo ricordarci infatti che lo sviluppo è parte del business, per molti versi lo stiamo facendo anche noi quando programmiamo.
+
+Viene piuttosto naturale capire che, il cosiddetto Time To Market (tempo di arrivo su un certo mercato), può risultare determinante nella riuscita di un prodotto; impatta per forza di cose anche su come il business lo percepisce perché determina quando inizieranno a tornare i profitti da quello che di fatto è investimento. In poche parole il business prende decisioni su questo parametro che diventa perno, insieme ad altri fattori, per delineare le strategie aziendali. Pianificare in base ad una stima temporale non è l'unica maniera di gestire questo aspetto, ma è sicuramente il preponderante che quindi merita di essere prima di tutto compreso come processo.
+
+Cercando di comprendere meglio la natura di quello che facciamo dobbiamo andare a definire cosa è il processo di produzione del software e quali incognite nasconde. Dal punto di vista della variabilità è delineabile come definito, statistico ed empirico; il tipo definito è il più semplice, non c'è alcuna sorpresa durante lo svolgimento, è qualcosa che abbiamo già fatto quindi non richiede alcuno sforzo perché sappiamo già prevederlo. L'esempio tipico è quello della configurazione di un servizio, l'installazione di un software e così via, non ha alcuna variabilità se non minima quindi qui la stima è deterministica e non comporta problematiche. Quello statistico è similare ma con variabilità più alta, qui si possono fare delle stime basate su esperienze precedenti, abbiamo una vasta gamma di pattern che si ripresentano e quindi predire un tempo di esecuzione è meno semplice del precedente ma sempre fattibile.
```

Comment:
Questi primi due o tre punti mi sembrano troppo "ricchi" dal punto di vista lessicale, ho dovuto leggerlo un paio di volte per capire ahah magari riscriverei l'inizio in maniera più semplice. "Dal punto di vista della variabilità è delineabile come definito, statistico ed empirico", per esempio, potrebbe diventare "Possiamo definire principalmente tre modalità di produzione del software: Definito, statistico ed empirico" o qualcosa di simile, cosa ne pensi?

Reply:
@Cadienvan ho chiarito i due concetti chiarendoli con qualche esempio, scorre meglio? Grazie!